### ISSUE TITLE
bufio.Scanner: token too long

### ISSUE THREAD

**Issue by ihard** [lomik/graphite-clickhouse#67]
I use graphite-clickhouse buuild from current master,
config:
```
[common]
listen = ":9090"
max-cpu = 8
memory-return-interval = "0s"
max-metrics-in-find-answer = 0

[clickhouse]
url = "http://127.0.0.1:8123/"
extra-prefix = ""
data-table = "graphite.points_cluster"
data-timeout = "1m0s"
rollup-conf = "/etc/graphite-clickhouse/graphite_rollup.xml"
index-table = "graphite.index_cluster"
index-use-daily = true
index-timeout = "1m"
tagged-table = "graphite.tagged_cluster"

[prometheus]
external-url = "https://grafana.example.com/prometheus"
page-title = "Prometheus Time Series Collection and Processing Server"

[[data-table]]
table = "graphite.points_cluster"
reverse = true
rollup-conf = "/etc/graphite-clickhouse/graphite_rollup.xml"

[[logging]]
logger = ""
file = "stdout"
level = "info"
encoding = "mixed"
encoding-time = "iso8601"
encoding-duration = "seconds"
```
graphite_rollup:
```
       <pattern>
            <regexp>^.*scrape_interval=10s.*$</regexp>
            <function>avg</function>
            <retention>
                <age>0</age>
                <precision>10</precision>
            </retention>
            <retention>
                <age>259200</age>
                <precision>60</precision>
            </retention>
            <retention>
                <age>2592000</age>
                <precision>600</precision>
            </retention>
        </pattern>
```

Collect 10s metric with telegraf and send to carbon-clickhouse.
If request metric for period above 7d - get error:
```
Object
xhrStatus:"complete"
request:Object
method:"GET"
url:"api/datasources/proxy/18/api/v1/query_range?query=mem_used_percent%7Bhost%3D%22metric%22%7D&start=1568832000&end=1569523200&step=600&timeout=10s"
response:Object
status:"error"
errorType:"execution"
error:"bufio.Scanner: token too long"
message:"bufio.Scanner: token too long"
```
in log graphite-clickhouse:
```
[2019-09-26T21:42:56.285+0300] INFO [query] query {"query": "SELECT splitByChar('=', Tag1)[2] as value FROM graphite.tagged_cluster WHERE (Tag1 LIKE 'host=%') AND (Date >= '2019-09-19') GROUP BY value ORDER BY value", "request_id": "81fe3ec423d58c03e75dccd575f9c19a", "time": 0.00851583}
[2019-09-26T21:42:56.285+0300] INFO access {"request_id": "81fe3ec423d58c03e75dccd575f9c19a", "grafana": "Org:1; Dashboard:; Panel:", "time": 0.008859073, "method": "GET", "url": "/api/v1/label/host/values", "peer": "[::1]:6660", "status": 200}
[2019-09-26T21:42:56.351+0300] INFO [query] query {"query": "SELECT Path FROM graphite.tagged_cluster  WHERE ((Tag1='__name__=mem_used_percent') AND (arrayExists((x) -> x='host=metric', Tags))) AND (Date >='2019-09-18' AND Date <= '2019-09-26') GROUP BY Path", "request_id": "5d67a79bb41a4e85e2d6c5eea1038326", "time": 0.011381311}
[2019-09-26T21:42:56.394+0300] INFO access {"request_id": "5d67a79bb41a4e85e2d6c5eea1038326", "grafana": "Org:1; Dashboard:400; Panel:3", "time": 0.054455356, "method": "GET", "url": "/api/v1/query_range?query=mem_used_percent%7Bhost%3D%22metric%22%7D&start=1568832000&end=1569523200&step=600&timeout=10s", "peer": "[::1]:6660", "status": 422}
```
Request only one metric, if request data, for example, 1 day -10 days ago, the data is displayed

**Comment by ihard** [comment]
fixed with #69

**Comment by ralphM78** [comment]
Hi,

I have seen this behavior with the new values of bufio.Scanner also. Sometimes a bigger time range is needed . Is it possible to generate a log entry if this size is to small and make the value configurable ?
